components/home/HomeCard.jsx

```diff
@@ -0,0 +1,79 @@
+import Link from 'next/link';
+
+import { useWindowSize } from '../../functions/hooks';
+import { LinkButton } from '../utils/Button';
+
+export function HomeCard({
+  alignRight,
+  children,
+  iconPrimary,
+  iconSecondary,
+  link,
+  linkText,
+  text,
+  titlePrimary,
+  titleSecondary
+}) {
+  const { windowSize } = useWindowSize();
+
+  let iconSize = 40;
+  if (windowSize >= 768) iconSize = 60;
+
+  return (
+    <div className="flex flex-col bg-darker mt-6 p-4 md:p-8 rounded-lg">
+      <div className="flex items-center mb-4 mx-2 lg:mx-4">
+        {alignRight && (
```

Comment:
I feel like you could use css to reverse the order of elements here rather than rendering the divs in different places, could make things simpler